src/arviz_plots/plots/tracedensplot.py

```diff
@@ -0,0 +1,241 @@
+        One or more variables to be plotted.
+        Prefix the variables by ~ when you want to exclude them from the plot.
+    filter_vars: {None, “like”, “regex”}, optional, default=None
+        If None (default), interpret var_names as the real variables names.
+        If “like”, interpret var_names as substrings of the real variables names.
+        If “regex”, interpret var_names as regular expressions on the real variables names.
+    sample_dims : iterable, optional
+        Dimensions to reduce unless mapped to an aesthetic.
+        Defaults to ``rcParams["data.sample_dims"]``
+    compact: bool, optional
+        Plot multidimensional variables in a single plot. Defaults to True
+    kind : {"kde", "hist", "dot", "ecdf"}, optional
+        How to represent the marginal density.
+    plot_collection : PlotCollection, optional
+    backend : {"matplotlib", "bokeh"}, optional
+    labeller : labeller, optional
+    aes_map : mapping, optional
+        Mapping of artists to aesthetics that should use their mapping in `plot_collection`
+        when plotted. Defaults to only mapping properties to the density representation.
+    plot_kwargs : mapping, optional
+        Valid keys are:
+
+        * trace -> passed to visuals.line
+        * divergence -> passed to visuals.line_x
+
+    pc_kwargs : mapping
+        Passed to :class:`arviz_plots.PlotCollection`
+
+    Returns
+    -------
+    PlotCollection
+    """
+    if sample_dims is None:
+        sample_dims = rcParams["data.sample_dims"]
+    if kind is None:
+        kind = rcParams["plot.density_kind"]
+    if dens_kwargs is None:
+        dens_kwargs = {}
+    if plot_kwargs is None:
+        plot_kwargs = {}
+    if pc_kwargs is None:
+        pc_kwargs = {}
+    else:
+        pc_kwargs = pc_kwargs.copy()
+
+    var_names = _var_names(var_names, dt.posterior.ds, filter_vars)
+
+    if var_names is None:
+        posterior = dt.posterior.ds
+    else:
+        posterior = dt.posterior.ds[var_names]
+
+    pc_kwargs.setdefault("aes", {"color": ["chain"]})
+
+    aux_dim_list = [dim for dim in posterior.dims if dim not in {"chain", "draw"}]
+
+    figsize, textsize, linewidth = scale_fig_size(
+        pc_kwargs.get("plot_grid_kws", {}).get("figsize", None),
+        rows=get_size_of_var(posterior, compact=compact),
+        cols=2,
+    )
+
+    plot_kwargs.setdefault("dens", {}).setdefault("lw", linewidth)
+    plot_kwargs.setdefault("trace", {}).setdefault("lw", linewidth)
+
+    if plot_collection is None:
+        if backend is None:
+            backend = rcParams["plot.backend"]
+        pc_kwargs.setdefault("cols", ["__column__"])
+        pc_kwargs.setdefault("plot_grid_kws", {"figsize": figsize})
+
+        if compact:
+            pc_kwargs.setdefault("rows", ["__variable__"])
+        else:
+            pc_kwargs.setdefault("rows", ["__variable__"] + aux_dim_list)
+
+        plot_collection = PlotCollection.grid(
+            posterior.expand_dims(__column__=2),
```

Comment:
```suggestion
            posterior.expand_dims(__column__=2).assign_coords(__column__=["dens", "trace"]),
```

do you think this would make the code more clear below? We'd have `coords={__column__="dens"/"trace"}` instead of `coords={__column__=0/1}`